scripts/gen_coveragefile_binja.py

```diff
@@ -0,0 +1,32 @@
+"""Generate a coverage file for WTF using Binary Ninja.
+
+@australeo - 2023
+"""
+import json
+import os
+from binaryninja import PluginCommand, basicblock, interaction
+
+def generate_coverage_file(bv):
+    # bv.file.filename: 'C:/path/to/binary.bndb'
+    name = bv.file.filename.split("/")[-1].split('.')[0]

```

Comment:
What do you think of changing that line w/ `Path(bv.file.filename).stem`?

Reply:
Done, good call!